package.json

```diff
@@ -28,11 +28,13 @@
     "cross-env": "^5.2.0",
     "eslint": "^5.3.0",
     "eslint-config-google": "^0.9.1",
+    "node-fetch": "^1.7.3",
```

Comment:
Summarizing changes here from in person discussion:
- `node-fetch`: look at using the existing `platform` abstraction
- `node-webcrypto-ossl`: it's dev-only for testing
- `idb`: see whether @shans is ok adding this (desired to make indexeddb code more readable)
- `jsrasign`: consider either adding a `platform` abstraction for this or embedding constant per comment elsewhere on this PR